lib/WP_Auth0_Routes.php

```diff
@@ -44,20 +54,32 @@ public function custom_requests( $wp ) {
 			$page = $wp->query_vars['pagename'];
 		}
 
+		$output = null;
 		if ( ! empty( $page ) ) {
 			switch ( $page ) {
 				case 'oauth2-config':
 					$this->oauth2_config();
 					exit;
 				case 'migration-ws-login':
-					$this->migration_ws_login();
-					exit;
+					$output = $this->migration_ws_login();
```

Comment:
What's going on here that you buffer it and then either echo it or return it? Only appears to be one case that needs to do this? Is the intention to have more since you broke out the  if (output) logic later on vs handling in the case?

Reply:
All these routes handle execution and, in some cases echo out a string, some just return. I'm combining the behavior into one flow so JSON is echoed and the process exits at the same time. In case the methods need to return instead of echoing, I added that `query_var` (mostly used for testing at this point but could be helpful in a custom API route).

Reply:
I still think the code could be tidier for this single use case. However, makes no real odds in terms of execution.

Reply:
All of these routes will do the same, just wanted to keep PRs a little more streamlined 👍